pages/positions/options.tsx

```diff
@@ -21,13 +21,13 @@ const Options: NextPage = () => {
       <div className={style.optionsContainer}>
         <h1 className={style.optionsHeader}>Open a New Position</h1>
         <div className={style.optionsButtonContainer}>
-          <Link href={'addliquidity'}>
+          <Link href={'deposit/0'}>
             <button className={style.optionsButtonContent}>
               <span className={style.hoveredOptionsButtonContent}></span>
               <div className={style.optionsButtonPrimary}>
-                <div className={style.buttonTitle}>Lend</div>
+                <div className={style.buttonTitle}>Provide Liquidity</div>
                 <div className={style.buttonText}>
-                  Provide liquidity with any ERC-20 Token pair.
+                  Lend liquidity with any ERC-20 Token pair.
```

Comment:
I think we need to change this to a more playful message. I think it's just saying the same thing as the title. Let's run this by Devin

Reply:
changed wording as per Devin